core/iwasm/interpreter/wasm_opcode.h

```diff
@@ -782,7 +782,9 @@ typedef enum WASMAtomicEXTOpcode {
 
 #define SET_GOTO_TABLE_ELEM(opcode) [opcode] = HANDLE_OPCODE(opcode)
 
-#if (WASM_ENABLE_JIT != 0 || WASM_ENABLE_FAST_INTERP != 0) \
+#if (WASM_ENABLE_JIT != 0                                           \
+     || (WASM_ENABLE_FAST_INTERP != 0 && defined(WASM_ENABLE_SIMDE) \
+         && WASM_ENABLE_SIMDE != 0))                                \
```

Comment:
Suggest to add `#ifndef WASM_ENABLE_SIMDE #define WASM_ENABLE_SIMDE 0 #endif` in core/config.h, so as to make WASM_ENABLE_SIMDE always defined, and then change this line to `|| (WASM_ENABLE_FAST_INTERP != 0 && WASM_ENABLE_SIMDE != 0)`